src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/MicrosoftTeamsOptions.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams;
+
+use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\MicrosoftTeamsActionInterface;
+use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\MicrosoftTeamsSection;
+use Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section\MicrosoftTeamsSectionInterface;
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ *
+ * @see https://docs.microsoft.com/en-us/outlook/actionable-messages/message-card-reference
+ */
+final class MicrosoftTeamsOptions implements MessageOptionsInterface
+{
+    private const MAX_ACTION = 4;
```

Comment:
```suggestion
    private const MAX_ACTIONS = 4;
```

plural please

Reply:
Ping, @OskarStark, should this be fixed?

Reply:
Indeed